exercises/practice/zebra-puzzle/zebra_puzzle.h

```diff
@@ -1,10 +1,11 @@
 #ifndef ZEBRA_PUZZLE_H
 #define ZEBRA_PUZZLE_H
 
-/// Determine the nationality of the resident who drinks water.
-const char *drinks_water(void);
+struct solution {
+   const char *drinks_water;
+   const char *owns_zebra;
+};
```

Comment:
```suggestion
typedef struct {
   const char *drinks_water;
   const char *owns_zebra;
} solution_t;
```

I think this would be more consistent with how we've handled structs elsewhere in the track